_episodes/036-luca-palmieri.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "From Zero to Production"
+date: 2021-09-10T15:30:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e036-luca-palmieri.mp3
+duration: "1:07:42"
+length: "65103280"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+#  From Zero to Production  
+
+Allen Wyma talks with Luke Palmieri, a principal engineer, also the book author of ["Zero To Production in Rust"](https://www.zero2prod.com/index.html)
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Highlights
+
+- "Zero To Production In Rust" is an introduction of backend development in Rust. 
+
+## Resources 
+- [Hexagonal Architecture](https://en.wikipedia.org/wiki/Hexagonal_architecture_(software))
+- [Asynchronous Programming in Rust](https://rust-lang.github.io/async-book/08_ecosystem/00_chapter.html)
+- [Rust for Rustaceans](https://nostarch.com/rust-rustaceans)
+- [Rust in Action](https://www.manning.com/books/rust-in-action)
```

Comment:
I'd prefer for these resources to be linked under the appropriate timed section below where possible. If a resource is referenced many times, then I'd be okay with listing it separately, though I'd prefer for the resources link list to be placed below the timestamps.